go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/grafana/xk6-browser
 go 1.20
 
 require (
-	github.com/chromedp/cdproto v0.0.0-20221023212508-67ada9507fb2
+	github.com/chromedp/cdproto v0.0.0-20240709201219-e202069cc16b
```

Comment:
I had to update the `chromedp` package as it didn't have the newest CDP protocol's type definitions and `Audits.issueAdded` events failed to unmarshal in `session.go`:

https://github.com/grafana/xk6-browser/blob/cd451af7be1ae457aecc9beae6412f3e0d281530/common/session.go#L85-L101

I simply updated to the newest version:

`go get -u github.com/chromedp/chromedp`